acceptance/apps/rails_test.go

```diff
@@ -90,8 +90,24 @@ var _ = Describe("RubyOnRails", func() {
 		Expect(err).ToNot(HaveOccurred(), out)
 
 		// Create a database for Rails
+		out, err = proc.Run("", false, "helm", "repo", "add", "bitnami", "https://charts.bitnami.com/bitnami")
+		Expect(err).ToNot(HaveOccurred(), out)
+
 		serviceName = catalog.NewServiceName()
-		out, err = env.Epinio("", "service", "create", serviceName, "mariadb", "10-3-22")
+		out, err = proc.Run("", false,
+			"helm", "install", serviceName, "bitnami/mariadb", "-n", rails.Org,
+			"--set", "auth.database=\"mydb\"",
```

Comment:
```suggestion
			"--set", `auth.database="mydb"`,
```

Less issue with quoted characters. See also following lines.

Reply:
Oh, I believe I see your issue and error for the test suite.

You have bash quoting here. The `"` around `mydb`, etc. are wrong.
Proper fix may be:
```suggestion
			"--set", "auth.database=mydb",
```
etc.

Reply:
The very fact that your commands are identical here and in the terminal is the issue I believe, and tripping you up.

Reply:
Out of band: Various ways of quoting, including the proposed, did not work